samples/matter/common/src/diagnostic/diagnostic_logs_crash.cpp

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "diagnostic_logs_crash.h"
+
+#include <lib/support/logging/CHIPLogging.h>
+
+#include <zephyr/kernel.h>
+
+using namespace Nrf;
+
+#define Verify(expr, value)                                                                                            \
```

Comment:
FYI there is `VerifyOrReturnValue` in Matter.

Reply:
Yes, I know, but I would not include Matter API here.